main.go

```diff
@@ -176,9 +177,26 @@ func mainWithExitCode() exitCode {
 		return failure("Failed to create container metadata provider: %v", err)
 	}
 
+	var extraSymbolEndpoints SymbolEndpoints
+	err = json.Unmarshal([]byte(args.extraSymbolEndpoints), &extraSymbolEndpoints)
+	if err != nil {
+		return failure("Unable to unmarshal extra symbol endpoints")
+	}
+
+	sites := extraSymbolEndpoints.getFieldString("Site")
+	apiKeys := extraSymbolEndpoints.getFieldString("ApiKey")
+	appKeys := extraSymbolEndpoints.getFieldString("AppKey")
+
+	sites = append(sites, args.site)
+	apiKeys = append(apiKeys, args.apiKey)
+	appKeys = append(appKeys, args.appKey)
```

Comment:
Verify that args.site, args.apiKey, args.appKey are not the empty string, before adding them to the list of symbol endpoints.